nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/nativeblas/Nd4jCpuPresets.java

```diff
@@ -137,6 +137,8 @@ public void map(InfoMap infoMap) {
                         .put(new Info("Nd4jPointer").cast().valueTypes("Pointer").pointerTypes("PointerPointer"))
                         .put(new Info("Nd4jLong").cast().valueTypes("long").pointerTypes("LongPointer", "LongBuffer",
                                         "long[]"))
+                        .put(new Info("Nd4jInt").cast().valueTypes("int").pointerTypes("IntPointer", "IntBuffer",
```

Comment:
We don't need Nd4jInt definition.

Reply:
After discussing with @saudet, this part is kept in, albeit Nd4jInt is typedef'ed to `int`

Reply:
Actually, let's not use a typedef at all. Just replace all instances of `Nd4jInt` in the code with `int`.

Reply:
Will do that then.